src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -960,21 +969,23 @@ public void verifyContainsElement(By by) {
     }
 
     /**
-     * Verifies the status message in the page is same as the one specified.
+     * Verifies the texts for status messages to the user in the page are equal to the expected texts.
```

Comment:
Let's change this to `Verifies that the texts of user status messages in the page are equal to the expected texts`.